### PR TITLE
Write reviews for book

### PR DESCRIPTION
**Pull request by awigdale**
#24 

### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---
_Your PR Notes Here_

write a review

### CODE REVIEW

File: server/api/middleware.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const AdminMW = (req, res, next) => {
   if (req.user && req.user.isAdmin) {
     next()
   } else {
+    console.log(req.user)
```
Comment (Kevingislason) [inline server/api/middleware.js]: Might want to delete

File: client/components/write-review.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    const bookId = this.props.selectedBook.id
+    const review = this.props.reviewForm.values
+    this.props.createReview(bookId, review)
+  }
+
+  render() {
+    return (
+      <Container textAlign="left" className="formcontainer">
+        <h4>Write a review for this book:</h4>
+        <Form onSubmit={this.handleSumbit.bind(this)}>
```
Comment (Kevingislason) [inline client/components/write-review.js]: If I remember right, there's no need for binding if you make handleSubmit an arrow function; might be a little neater

File: server/api/reviews.spec.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ describe('Review routes', () => {
       stars: 5
     }
     const reviewData2 = {
-      title: 'I liked it',
+      title: 'I loved it',
```
Comment (Kevingislason) [inline server/api/reviews.spec.js]: Shouldn't the two reviews have different titles to distinguish them?

File: server/db/models/reviews.spec.js [hunk 4]
```diff
@@ -3,9 +3,9 @@ const db = require('..')
 const Review = db.model('review')
 
 describe('Review model', () => {
-  beforeEach(() => {
-    return db.sync({force: true})
-  })
+  // beforeEach(() => {
+  //   return db.sync({force: true})
+  // })
```
Comment (tomasgnr) [inline server/db/models/reviews.spec.js]: Abby, is this supposed to be commented? please add a note if so.

**Pull request merged**